client/maisie/interface.py

```diff
@@ -10,8 +10,15 @@
 
 
 @click.group()
-def cli():
-    pass
+@click.option(
+    "--output_json", is_flag=True, help="Display output in pure json instead of a table"
```

Comment:
I am still wondering if this is the right approach. What would happen if we wanted to add an additional way of formatting the response in the future? What would happen if I were to include both `--output_json` and `--output_xml` in the same time?

What if instead of creating multiple global options for formatting, we stick to the regular approach, meaning having a single option such as `--output=json`? It seems like that would easily allow us the extend the functionality when needed.
Your input on this issue would be most welcome.